### DISCUSSION TITLE
"Default Link Type" setting for Link fields

### DISCUSSION THREAD

**Discussion by mmikkel** [craftcms/cms#15516] [Ideas] [answered]
It'd be great if it was possible to set a _default link type_ in the field settings for Link fields.

Currently, empty Link fields will always be pre-selected to the first allowed link type (typically; "URL"). However, that link type isn't necessarily the one authors want to use _most of the time_, for any given field. The ability to have a specific Link field default to (for example) "Entry", whilst still having "URL" and other types available, would be helpful in reducing AX friction as much as possible.

**Comment by droemmelbert** [comment]
Any updates on this @brandonkelly ? Highly anticipating this feature!

**Comment by maxstrebel** [comment]
I want to add my +1. Added it to a new client site and they added each link as URL instead of using the Entry Relation (although I added a BOLD info text to please use the Entry if possible).

**Comment by brandonkelly** [comment, marked as answer]
We’ve made the Allowed Link Types setting sortable for Craft 5.5 (#15963). Empty Link fields will always default to whichever link type is listed first :tada:

**Reply by brandonkelly** [reply]
Craft 5.5.0 is out with that change 🎉
